apps/src/StudioApp.js

```diff
@@ -469,7 +469,8 @@ StudioApp.prototype.init = function (config) {
     Blockly.mainBlockSpaceEditor.addUnusedBlocksHelpListener(function (e) {
       utils.showUnusedBlockQtip(e.target);
     });
-    Blockly.mainBlockSpaceEditor.addChangeListener(_.bind(function () {
+    // Store result so that we can cleanup later in tests
+    this.changeListener = Blockly.mainBlockSpaceEditor.addChangeListener(_.bind(function () {
```

Comment:
Though that's probably fine, I'm somewhat wary of making any changes without knowing enough about the code to test it correctly.